CHANGES.rst

```diff
@@ -5,6 +5,9 @@
 Unreleased
 ==========
 
+2018/04/16 0.1.2
```

Comment:
think this should be 0.2 as its adding a new feature

Reply:
ok!